spec/authn_spec.rb

```diff
@@ -1,7 +1,11 @@
+require 'spec_helper'
+
 require 'conjur/authn'
 require 'conjur/config'
 
 describe Conjur::Authn do
+  include_context "fresh config"
```

Comment:
This is neat! I don't understand how it all works, but maybe I could learn some time.